Atomic/atomic.py

```diff
@@ -379,8 +379,18 @@ def get_fq_name(self, image_info):
             if self.image in image_info['RepoTags']:
                 return self.image
 
+            possibles = []
+            for i in image_info['RepoTags']:
+                try:
+                    possibles.append(self.get_fq_image_name(i))
+                except RegistryInspectError:
+                    possibles.append(None)
```

Comment:
Wouldn't this be a strange case where an Image would throw an exception on get_fq_image_name?  Would something be wrong in their database or a bug in code?  I think we should throw a different exception, so that it would be easier to debug.